src/test/java/TestClass.java

```diff
@@ -0,0 +1,17 @@
+import org.junit.BeforeClass;
```

Comment:
This seems to be in the default package. This is not really done even in test systems. You should put this test in its own package.